mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -0,0 +1,87 @@
+pipeline {
+    agent {
+        docker {
+            image 'rocm/mlir:rocm4.1-latest'
+            args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+            label 'mlir'
+            alwaysPull true
+        }
+    }
+    stages {
+        stage('Environment') {
+            steps {
+                sh 'cat /etc/os-release'
+                sh '/opt/rocm/bin/rocm-smi'
+            }
+        }
+        stage('Build MIOpen') {
```

Comment:
Might I suggest replacing these with the sort of cmakeBuild calls and other such things that I'm using is the new system and that have landed for the MIOpen test? This'll make it easier to integrate (and not accidentally break) your changes when the One Big Jenkinsfile lands. 

I can help explain what I've put together.

Reply:
I'll change the Jenkinsfile to use cmakeBuild after your "revert of revert 241" is in.

Reply:
Makes sense - what are we waiting on with that revert? I know there's currently a lot of performance-critical work going in and so breaking the build is a pretty bad move and thus we may want to wait until ... next week?

Reply:
However, that PR landing converts existing builds to the newer approach - everything that might block you from using that setup has already landed. (and it's being successfully used in the `.miopen-test` file, last I checked)

Reply:
I think wait until we're on a stable state, so it could be as soon as tomorrow, but please talk to Jack first to make sure that important PRs are not blocked.

Reply:
Fundamentally, my CI refactor is, at this point (now that the libMLIR and CMake changes are merged in) independent of this, so please take the `cmakeBuild` and such as a style suggestion. If that change isn't made, I can do it in my PR.